src/platforms/windows/daemon/windowsfirewall.cpp

```diff
@@ -7,6 +7,8 @@
 #include "leakdetector.h"
 #include "../windowscommons.h"
 #include "../../daemon/interfaceconfig.h"
+#include "../../ipaddressrange.h"
```

Comment:
You don't need the `../../` in the include directives since `src/` is in the search path.